tianshou/env/venvs.py

```diff
@@ -98,7 +76,11 @@ def __init__(self,
         self.waiting_id = []
         # all environments are ready in the beginning
         self.ready_id = list(range(self.env_num))
-        self.unique_id = generate_id()
+        self.is_closed = False
+
+    def _assert_is_closed(self):
```

Comment:
Func name is ``_assert_is_closed ``, but the assertion is ``assert not self.is_closed``

Reply:
```suggestion
    def _assert_is_not_closed(self):
```